### PR TITLE
feat: improve and extend 🇫🇷 fr and 🇨🇭fr_CH locales

### PR DESCRIPTION
**Pull request by Yago**
Adds a lot of locales to fr and fr_CH with a focus on inclusive words; mostly for nouns and adjectives.

### CODE REVIEW

File: src/locales/fr/animal/cow.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  'Bazadaise',
+  'Béarnaise',
+  'Bétizu',
+  'Bleue du Nord',
+  'Bordelaise (Nouvelle)',
+  'Braunvieh',
+  'Brava',
+  'Bretonne pie noir',
+  'Brune',
+  'Brune',
```
Comment (ST-DDT) [inline src/locales/fr/animal/cow.ts:15]: This one seems to be duplicate.

File: src/locales/fr/animal/lion.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default [
+  "lion de l'Atlas",
+  'lion du Cap',
+  "Lion d'Afrique",
+  "lion d'Asie",
```
Comment (ST-DDT) [inline src/locales/fr/animal/lion.ts:5]: Is the difference in capitalization intentional?

File: src/locales/fr/word/interjection.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
+export default [
+  'ah',
+  'aïe',
+  'areu areu',
+  'atchoum',
+  'badaboum',
+  'bang',
+  'bang',
+  'bang',
```
Comment (ST-DDT) [inline src/locales/fr/word/interjection.ts:9]: These words are duplicate.

File: src/locales/fr/word/interjection.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  'ha',
+  'ha ha',
+  'hé',
+  'hi',
+  'meuh',
+  'miam',
+  'miaou',
+  'oh',
+  'ouah',
+  'ouah',
```
Comment (ST-DDT) [inline src/locales/fr/word/interjection.ts:41]: Same here

File: src/locales/fr_CH/animal/lion.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export default [
+  "lion de l'Atlas",
+  'lion du Cap',
+  "Lion d'Afrique",
+  "lion d'Asie",
```
Comment (ST-DDT) [inline src/locales/fr_CH/animal/lion.ts:5]: Is the difference in capitalization intentional?

File: src/locales/fr_CH/company/bs_adjective.ts [hunk 6]
```diff
@@ -0,0 +1,67 @@
+export default [
+  'clicks-and-mortar',
+  'value-added',
+  'vertical',
+  'proactive',
```
Comment (ST-DDT) [inline src/locales/fr_CH/company/bs_adjective.ts:5]: This file looks like `en` to me.

File: src/locales/fr_CH/company/bs_noun.ts [hunk 7]
```diff
@@ -0,0 +1,46 @@
+export default [
+  'synergies',
+  'web-readiness',
+  'paradigms',
+  'markets',
```
Comment (ST-DDT) [inline src/locales/fr_CH/company/bs_noun.ts:5]: same here

File: src/locales/fr_CH/company/bs_verb.ts [hunk 8]
```diff
@@ -0,0 +1,62 @@
+export default [
+  'implement',
+  'utilize',
+  'integrate',
+  'streamline',
```
Comment (ST-DDT) [inline src/locales/fr_CH/company/bs_verb.ts:5]: And here

File: src/locales/fr_CH/company/adjective.ts [hunk 9]
```diff
@@ -0,0 +1,102 @@
+export default [
+  'Adaptive',
+  'Advanced',
+  'Ameliorated',
+  'Assimilated',
```
Comment (ST-DDT) [inline src/locales/fr_CH/company/adjective.ts:5]: This looks like english

File: src/locales/fr_CH/company/descriptor.ts [hunk 10]
```diff
@@ -0,0 +1,103 @@
+export default [
+  '24 hour',
+  '24/7',
+  '3rd generation',
+  '4th generation',
+  '5th generation',
+  '6th generation',
+  'actuating',
+  'analyzing',
+  'asymmetric',
```
Comment (ST-DDT) [inline src/locales/fr_CH/company/descriptor.ts:10]: And here

File: src/locales/fr_CH/company/noun.ts [hunk 11]
```diff
@@ -0,0 +1,106 @@
+export default [
+  'ability',
+  'access',
+  'adapter',
+  'algorithm',
```
Comment (ST-DDT) [inline src/locales/fr_CH/company/noun.ts:5]: And here

File: src/locales/fr_CH/word/interjection.ts [hunk 12]
```diff
@@ -0,0 +1,67 @@
+export default [
+  'ah',
+  'aïe',
+  'areu areu',
+  'atchoum',
+  'badaboum',
+  'bang',
+  'bang',
+  'bang',
```
Comment (ST-DDT) [inline src/locales/fr_CH/word/interjection.ts:9]: These words are duplicate.

File: src/locales/fr_CH/word/interjection.ts [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+  'ha',
+  'ha ha',
+  'hé',
+  'hi',
+  'meuh',
+  'miam',
+  'miaou',
+  'oh',
+  'ouah',
+  'ouah',
```
Comment (ST-DDT) [inline src/locales/fr_CH/word/interjection.ts:41]: Same here

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. This is quite a lot to review. It will probably take a few days to review it in more detail.

File: src/locales/fr/animal/lion.ts (hunk 2 above)
Comment (Yago) [reply src/locales/fr/animal/lion.ts]: No, thank you for the review.

File: src/locales/fr_CH/company/adjective.ts (hunk 9 above)
Comment (Yago) [reply src/locales/fr_CH/company/adjective.ts]: It came from the fr/company source, which was already in English. I removed it, thank you for the review.

**Comment by Yago** [comment]
@ST-DDT Thank you for your feedbacks, I fixed everything.

**Review by ST-DDT — APPROVED** [review]
I checked the translations using Google Translate.
Looks good to me.

**Comment by ST-DDT** [comment]
Please rebase/merge main.

**Comment by Yago** [comment]
@ST-DDT I clean the merge commit from the branch and redid it properly 😉

File: src/locales/fr_CH/address/street_name.ts [hunk 14]
```diff
@@ -0,0 +1 @@
+export default ['{{address.street_prefix}} {{address.street_suffix}}'];
```
Comment (ST-DDT) [inline src/locales/fr_CH/address/street_name.ts:1]: These are not street names. 
We fixed some inconsistent file names/contents in all locales, so this one no longer matches.
Please either copy them over from fr again or put some real street names in here.
Sorry for the trouble.

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: src/locales/fr_CH/address/street_name.ts (hunk 14 above)
Comment (Yago) [reply src/locales/fr_CH/address/street_name.ts]: No worries. It's fixed.

**Review by Shinigami92 — APPROVED** [review]

**Review by pkuczynski — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Review by import-brain — APPROVED** [review]

**Pull request merged**